.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+[ ] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/tdms-parser/blob/master/CONTRIBUTING.md).
```

Comment:
This will make it look like a real checkbox
```suggestion
- [ ] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/tdms-parser/blob/master/CONTRIBUTING.md).
```